docs/rfc/2022-07-05-mo-cluster.md

```diff
@@ -134,6 +174,37 @@ sequenceDiagram
 The Operator is not going to manage object storage for users in the first version.
 Documentations will be provided to guide users to setup a S3 compatible object storage (e.g. S3, minio) for the mo-cluster.
 
+Credentials for the application process to access S3 will be injected by the Operator. Official AWS SDK defines a well-known priority order to discover different credential source from the environment, e.g. EC2 instance, indentity federation, environment variables and credential configs.
+The Operator will allow users to specify the following credential sources:
+
+1. EC2 instance meta: no credential for DB process, the DB process will automatically discover whether it is running in an AWS EC2 instance and use the role of the instance to access the bucket. Only applicable for AWS S3.
+2. Web identity: inject a identity token file to the container of the DB process and expose the file path of the token through environment variable, the DB process will automatically discover the token file, exchange temporary AWS credentials with this token and periodically refresh the credentials in memory. Only applicable for AWS S3;
+3. Access key: user create a k8s secret contains a pair of static access key and secret key then reference that secret when creating the mo-cluster. The access key and secret key will be injected to the environment variables of DB containers by the Operator. Applicable for all S3 compatible storage system including AWS S3 and minio.
```

Comment:
3. Access key: the user creates a k8s secret containing keypair <AccessKey, SecretKey> and then references that secret when creating the mo-cluster. The access key and secret key will be injected into the environment variables of DB containers by the Operator. Applicable for all S3 compatible storage systems including AWS S3 and Minio.